frontend/src/components/atoms/NumberInputField/NumberInputField.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/require-default-props */
+import React, { Component } from 'react';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Form, InputNumber } from 'antd';
+import PropTypes from 'prop-types';
+
+import './NumberInputField.css';
+
+class NumberInputField extends Component {
+  render() {
+    const {
+      max,
+      min,
+      label,
+      title,
+      name,
+      handleInputChange,
+    } = this.props;
+    return (
+      <Form.Item
+        label={label}
+        name={name}
+        tooltip={{
+          title,
+          icon: <InfoCircleOutlined />,
+        }}
+        onChange={handleInputChange}
+      >
+        <InputNumber
+          max={max}
+          min={min}
+        />
+      </Form.Item>
+    );
+  }
+}
+
+NumberInputField.propTypes = {
+  max: PropTypes.number,
+  min: PropTypes.number,
+  label: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  handleInputChange: PropTypes.func.isRequired,
+};
+
```

Comment:
Rather than disabling this warning, you could just add the default props like this:

```
NumberInputField.defaultProps = {
    max: 100,
    ....
};
```

https://stackoverflow.com/questions/57009097/eslint-react-prop-types-vs-react-require-default-props

Reply:
Done (though setting 'min' and 'max' defaults to the generic 'NumberInputField' can potentially create issues later).